test/bdd/pkg/edv/edv_steps.go

```diff
@@ -49,71 +73,325 @@ func (e *Steps) createDataVault(vaultID, expectedVaultLocation string) error {
 	}
 
 	if vaultLocation != expectedVaultLocation {
-		return fmt.Errorf("expected the new data vault location to be %s, got %s instead",
-			expectedVaultLocation, vaultLocation)
+		return unexpectedValueError(expectedVaultLocation, vaultLocation)
 	}
 
 	return nil
 }
 
-func (e *Steps) edvServerRunningWithDataVault(vaultID string) error {
-	err := e.createDataVault(vaultID, "localhost:8080/encrypted-data-vaults/"+vaultID)
+func (e *Steps) clientConstructsAStructuredDocument(docID string) error {
+	meta := make(map[string]interface{})
+	meta["created"] = "2020-01-10"
+
+	content := make(map[string]interface{})
+	content["message"] = "In Bloc we trust"
+
+	e.bddContext.StructuredDocToBeEncrypted = operation.StructuredDocument{
+		ID:      docID,
+		Meta:    meta,
+		Content: content,
+	}
+
+	return nil
+}
+
+func (e *Steps) clientEncryptsTheStructuredDocumentAndUsesItToConstructAnEncryptedDocument() error {
```

Comment:
it can be just called `clientEncryptsTheStructuredDocument` :)

Reply:
Done